### PR TITLE
updates URL capitalization

### PR DESCRIPTION
**Pull request by emileighoutlaw**
from fec.gov to FEC.gov

[ci skip]

### CODE REVIEW

File: templates/layouts/main.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
   <header class="site-header">
     <div class="disclaimer">
-      <span class="disclaimer__left">Looking for <a href="http://www.fec.gov">fec.gov?</a></span>
+      <span class="disclaimer__left">Looking for <a href="http://www.fec.gov">FEC.gov?</a></span>
```
Comment (noahmanger) [inline templates/layouts/main.html]: I noticed our content guide now recommends all lowercase urls. Thoughts?

Comment (emileighoutlaw) [inline templates/layouts/main.html]: Excellent note. And so glad you're reading our content guide!

We've been using `FEC.gov` and `beta.FEC.gov` in our other writings. But you're correct that it isn't aligned with our (well reasoned, well supported)18F style of always using lowercase. 

We *can* switch to lowercase (for consistency), but I think we'll need to change all our references to `fec.gov` or `beta.fec.gov`. 

@leahbannon — thoughts on this?

Comment (noahmanger) [inline templates/layouts/main.html]: I'll merge this for now to keep things consistent, and then if we want to make a universal change we can.

**Comment by emileighoutlaw** [comment]
Sounds good. I think this is something we should talk about, and I'm glad
you brought it up.

**Pull request merged**